llvm/lib/Target/X86/AsmParser/X86AsmParser.cpp

```diff
@@ -48,6 +48,10 @@ static cl::opt<bool> LVIInlineAsmHardening(
     cl::desc("Harden inline assembly code that may be vulnerable to Load Value"
              " Injection (LVI). This feature is experimental."), cl::Hidden);
 
+static cl::opt<bool> AsmOptimize("x86-inline-asm-optimize", cl::init(true),
```

Comment:
It's not inline asm opt. It works on assembly.

Reply:
clang format

Reply:
Done.